routes/drones.js

```diff
@@ -3,34 +3,68 @@ const router = express.Router();
 
 // require the Drone model here
 
-router.get('/drones', (req, res, next) => {
-  // Iteration #2: List the drones
-  // ... your code here
-});
+const Drone = require("../models/Drone.model");
+
+router.get('/drones', async (req, res, next) => {
+ try {
+  let drones = await Drone.find();
 
-router.get('/drones/create', (req, res, next) => {
-  // Iteration #3: Add a new drone
-  // ... your code here
+  res.render("drones/list", {drones});
+  
+ } catch (error) {
+    console.log(error)
+    next(error)
+
+ }
 });
 
-router.post('/drones/create', (req, res, next) => {
-  // Iteration #3: Add a new drone
-  // ... your code here
+router.get('/drones/create', (req, res, next) => res.render("drones/create-form"));
```

Comment:
You have a `post` and a `get` route to this `/drones/create` but you don't have in your website any link or button for you to proceed that action.